testproxy/services/read-modify-write-row.js

```diff
@@ -34,7 +34,7 @@ const readModifyWriteRow = ({clientMap}) =>
     const table = getTableInfo(bigtable, tableName);
     const row = table.row(handWrittenRequest.id);
     try {
-      const result = await row.createRules(handWrittenRequest.rules);
+      const [result] = await row.createRules(handWrittenRequest.rules);
```

Comment:
The return value here is actually an array where the first element is the results so this change is necessary.